<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Command_Panel</class>
 <widget class="QDialog" name="Command_Panel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox_target">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>171</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>Measurement Target</string>
   </property>
   <widget class="QRadioButton" name="radioButton_current">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Current</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_voltage">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Voltage</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_resistance">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Resistance</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_save_file">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>441</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="label_file_path">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>File path:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_file_path">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>19</y>
      <width>271</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_browse">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>14</y>
      <width>81</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
  </widget>
  <widget class="QCheckBox" name="checkBox_save_file">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>331</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Save measurement to file</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_type">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>10</y>
     <width>251</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>Measurement Type</string>
   </property>
   <widget class="QRadioButton" name="radioButton_single_data">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>31</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Single data</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_periodic_sampling">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>61</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Periodic sampling</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_sampling_period">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>92</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Sampling period:</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox_sampling_period">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>90</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_second">
    <property name="geometry">
     <rect>
      <x>215</x>
      <y>92</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>s</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
